### DISCUSSION TITLE
Theory for pyrolysis in FDS Verification Guide

### DISCUSSION THREAD

**Discussion by lihy213** [firemodels/fds#12604] [Q&A]
Dear contributors,

In the **FDS Verification Guide**, **page 171**, it shows that "_For cylindrical surfaces, the volume per unit surface area is r/2_", if it means the top and bottom surface of a cylinder are not included, or the result should be $\pi r^2h\over {2\pi r^2+2\pi rh}$, h is the height of the cylinder.

Kind regards,
LI

**Comment by drjfloyd** [comment]
Ends are not included
V = pi r^2 h
A = 2 pi r h
V/A = r/2

**Reply by lihy213** [reply]
Thanks! But, could you please explain it? 😄

**Comment by mcgratta** [comment]
Two reasons -- first, the area of the ends is usually much smaller than the area of the side. Second, the assumptions about radiative and convective heat transfer are based on the assumption that the length is significantly longer than the diameter. If that assumption is not appropriate for you, you may want to model the particle as a sphere.

**Reply by lihy213** [reply]
Thanks for your kind reply! Now it becomes clear. 😄
